src/makeRobot.js

```diff
@@ -38,7 +38,60 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
```

Comment:
```suggestion
    },
    
    get info() {
```
добавляй пробел между методами для читабельности